simplipy/sensor.py

```diff
@@ -74,6 +74,13 @@ def low_battery(self) -> bool:
     def settings(self) -> bool:
         """Return the sensor's settings."""
         return self.sensor_data['setting']
+    
+    @property
+    def entrystatus(self) -> str:
+        if self._type == SensorTypes.entry:
```

Comment:
Please follow [the same pattern that is used for temperature sensors in V3 systems](https://github.com/bachya/simplisafe-python/blob/01fdd25d28d9f1decab7011ec7499982d7e69f0c/simplipy/sensor.py#L118-L125).